functions/get_rsan_ip.pp

```diff
@@ -0,0 +1,14 @@
+function rsan::get_rsan_ip() {
+  if $settings::storeconfigs {
+    $rsan_ip =
+                puppetdb_query('resources[certname] {
+                    type = "Class" and
+                    title = "Rsan::Importer" and
+                    nodes {
+                      deactivated is null and
+                      expired is null
+                    }
+                  }').map |$data| { $data['certname'] }
```

Comment:
use `}').map |$data| { $data['value'] }` so that it uses the correct field output, as the result of the query looks like this:
```
[
  {
    "certname": "rsan.dreamworx.nl",
    "environment": "production",
    "name": "ipaddress",
    "value": "192.168.250.234"
  }
]
```